### ISSUE TITLE
Trim options not working.

### ISSUE THREAD

**Issue by zaxwebs** [duniul/trim-image-data#2]
This bit from docs doesn't seem to work. Also, wouldn't it be better to be able to pass in an RGBA array [255, 255, 255, 255] instead of a callback?
```
// trim surrounding white pixels
const trimmedWhite = trimImageData(imageData, {
  trimColor: ({ red, green, blue, alpha }) => {
    return red === 255 && green === 255 && blue === 255 && alpha === 255;
  };
});
```

**Comment by duniul** [comment]
Thanks for reporting this! 🙏 Sorry for the slow response, must have missed the notifications for this one!

> This bit from docs doesn't seem to work.

You're right! For it to actually trim pixels it would have to check `red === 255 && green === 255 && blue === 255 && alpha === 255`, which is very unintuitive. 🤦

In fact, that option is so confusing that I've released a new major version that changes the behavior to what the name of the option implies. If `true` is returned then that means we want to trim that color. No idea why I didn't implement it like that in the first place.

> Also, wouldn't it be better to be able to pass in an RGBA array [255, 255, 255, 255] instead of a callback?

The reason for the callback is so that you do dynamic conditions, like checking ranges of colors instead of exact pixels. That said, a shorthand would be good for those use cases! Added in the last release 👍 

[See v2.0.0 for the changes.](https://github.com/duniul/trim-image-data/releases/tag/v2.0.0)
